deploy/olm-catalog/ibm-block-csi-operator/ibm-block-csi-operator.v1.2.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,679 @@
+    support: IBM
+    alm-examples: >-
+      [
+        {
+          "apiVersion": "csi.ibm.com/v1",
+          "kind": "IBMBlockCSI",
+          "metadata": {
+            "name": "ibm-block-csi"
+          },
+          "spec": {
+            "controller": {
+              "repository": "ibmcom/ibm-block-csi-driver-controller",
+              "tag": "1.2.0",
+              "imagePullPolicy": "IfNotPresent",
+              "affinity": {
+                "nodeAffinity": {
+                  "requiredDuringSchedulingIgnoredDuringExecution": {
+                    "nodeSelectorTerms": [
+                      {
+                        "matchExpressions": [
+                          {
+                            "key": "kubernetes.io/arch",
+                            "operator": "In",
+                            "values": [
+                              "amd64",
+                              "s390x"
+                            ]
+                          }
+                        ]
+                      }
+                    ]
+                  }
+                }
+              }
+            },
+            "node": {
+              "repository": "ibmcom/ibm-block-csi-driver-node",
+              "tag": "1.2.0",
+              "imagePullPolicy": "IfNotPresent",
+              "affinity": {
+                "nodeAffinity": {
+                  "requiredDuringSchedulingIgnoredDuringExecution": {
+                    "nodeSelectorTerms": [
+                      {
+                        "matchExpressions": [
+                          {
+                            "key": "kubernetes.io/arch",
+                            "operator": "In",
+                            "values": [
+                              "amd64",
+                              "s390x"
+                            ]
+                          }
+                        ]
+                      }
+                    ]
+                  }
+                }
+              }
+            },
+            "sidecars": [
+              {
+                "name": "csi-node-driver-registrar",
+                "repository": "registry.redhat.io/openshift4/ose-csi-driver-registrar",
+                "tag": "latest",
+                "imagePullPolicy": "IfNotPresent"
+              },
+              {
+                "name": "csi-provisioner",
+                "repository": "registry.redhat.io/openshift4/ose-csi-external-provisioner-rhel7",
+                "tag": "latest",
+                "imagePullPolicy": "IfNotPresent"
+              },
+              {
+                "name": "csi-attacher",
+                "repository": "registry.redhat.io/openshift4/ose-csi-external-attacher",
+                "tag": "latest",
+                "imagePullPolicy": "IfNotPresent"
+              },
+              {
+                "name": "csi-snapshotter",
+                "repository": "registry.access.redhat.com/openshift4/ose-csi-external-snapshotter-rhel7",
```

Comment:
please use the [correct registry hostname](https://github.com/IBM/ibm-block-csi-operator/pull/69#discussion_r415096646)